package.json

```
@@ -9,10 +9,15 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@radix-ui/react-slot": "^1.1.1",
+    "class-variance-authority": "^0.7.1",
```

```suggestion
    "@radix-ui/react-slot": "^1.1.1",
    "class-variance-authority": "^0.7.1",
```
delete this useless dependencies